apps/testingaccessibility/src/pages/api/sanity/webhooks/add-all-records-to-algolia.ts

```diff
@@ -4,9 +4,7 @@ import {sanityAlgolia} from 'utils/algolia'
 import {setupHttpTracing} from '@vercel/tracing-js'
 import {withSentry} from '@sentry/nextjs'
 import {tracer} from 'utils/honeycomb-tracer'
-import get from 'lodash/get'
-
-const secret = process.env.NEXT_PUBLIC_ALGOLIA_API_WRITE_KEY as string
```

Comment:
the algolia write key was being used in the sanity request validator which as far as I can tell requires the Sanity webhook secret which wasn't in the environment variables so it's added below

🚨 also note: **private keys should never be prefixed with `NEXT_PUBLIC`**.